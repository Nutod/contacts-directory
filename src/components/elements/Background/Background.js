import React from "react";
import styled from "styled-components";
import BackgroundImage from "../../../assets/images/background.jpg";

const BackgroundContainer = styled.div`
	height: 50vh;
	width: 50vw;
`;

export default function Background() {
	return (
		<BackgroundContainer>
			<img src={BackgroundImage} alt="" style={{ width: "100%" }} />
		</BackgroundContainer>
	);
}
