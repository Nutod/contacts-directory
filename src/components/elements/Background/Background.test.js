import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import Background from './Background'

Enzyme.configure({ adapter: new Adapter() });

describe("<Background />", () => {
  const wrapper = shallow(<Background />);

  it("Should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
