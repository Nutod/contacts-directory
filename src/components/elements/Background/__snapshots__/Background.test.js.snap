// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> Should match snapshot 1`] = `
<img
  alt=""
  src="background.jpg"
  style={
    Object {
      "width": "100%",
    }
  }
/>
`;
